lib/tasks/sumofus.rake

```diff
@@ -2,50 +2,146 @@ require 'open-uri'
 
 namespace :sumofus do
   desc 'Import legacy actions into your database from a specified file'
-  task seed_legacy_actions: :environment do
-    if ARGV[1].nil?
+  task :seed_legacy_actions, [:action_file, :page_img_file, :follow_img_file] => :environment do |task, args|
+
+    if args[:action_file].blank?
       abort('Requires a valid url to a file containing the legacy actions to seed.')
+    else
+      page_image_handle = open(args[:page_img_file])
     end
 
-    if ARGV[2].nil?
+    if args[:page_img_file].blank?
       abort('Requires a valid url to a file containing a default header image to attach to the page.')
+    else
+      puts "Loading page data"
+      page_data_handle = open(args[:action_file])
+      page_data = JSON.load(page_data_handle.read)
+      page_data_handle.close
+      puts "Page data loaded"
+    end
+
+    if args[:follow_img_file].blank?
+      follow_image_handle = open(args[:page_img_file])
+    else
+      follow_image_handle = open(args[:follow_img_file])
     end
 
-    file_handle = open(ARGV[1])
-    data = file_handle.read
-    file_handle.close
+    def create_post_action_pages(layout_id, image_handle)
+      pages = {}
+      ['en', 'fr', 'de'].map do |locale|
+        page = Page.find_or_initialize_by(title: I18n.t('fundraiser.generic.title', locale: locale))
+        page.liquid_layout_id = layout_id
+        page.language_id = language_ids[locale]
+        page.content = I18n.t('fundraiser.generic.body', locale: locale)
+        page.save!
+        page.images.create!(content: image_handle) if page.images.empty?
+        pages[locale] = page
+      end
+      pages
+    end
 
-    image_handle = open(ARGV[2])
+    def manage_newlines(content)
```

Comment:
It's a bunch of different WYSIWYGs spitting out newline formatting in different ways. This standardizes the ones that I came across into something reasonable.